contrib/search/confluence/ConfluenceCollator

```diff
@@ -0,0 +1,87 @@
+import { DocumentCollator } from '@backstage/search-common';
```

Comment:
Perhaps worth adding a `.ts` to the filename here?  Could make it easier to ready (syntax highlighting) when browsing.

Reply:
Hey @iamEAP! Good call, I actually originally had the file types included but I repeatedly ran into an eslint error 
`  1:1  error  Parsing error: The keyword 'import' is reserved`
I tried doing a little debugging and updating my .eslintrc.js file but unfortunately couldn't get it work and don't have time to look into it further

Reply:
Let me know if there is a known fix for this, if so I'd be happy to update

Reply:
From taking a quick look, it looks like the contrib folder is mostly markdown files with code snippets

So maybe another workaround is having all the code as code snippets in a markdown file? You can then add the name of the files as part of the markdown file in the example

Reply:
Thanks for the callout - updated to markdown files!